ghga_connector/cli.py

```diff
@@ -165,3 +165,44 @@ def download(  # pylint: disable=too-many-arguments
                 submitter_public_key=submitter_public_key,
             )
         file_stager.update_staged_files()
+
+
+@cli.command()
+def decrypt(
+    *,
+    input_file: Path = typer.Option(
+        ..., help="Path to the file that should be decrypted."
+    ),
```

Comment:
Maybe allow this to be a single file or a directory with multiple files.

Reply:
Changed to list of paths, changed output_path to output_dir defaulting to cwd and added check so we don't overwrite anything by accident